tox.ini

```diff
@@ -9,7 +9,7 @@ deps = -r{toxinidir}/test-requirements.txt
        -r{toxinidir}/requirements.txt
 commands =
    python -V
-   py.test -vvv -s --ignore=kubernetes/e2e_test
+   py.test -vvv -s --showlocals --ignore=kubernetes/e2e_test
```

Comment:
Does this allow us to see the variable values (e.g. the `timeout`) in the test (when the test fails)? I like this

Reply:
yup, that's exactly why I did this change

Reply:
but while looking at the logs in CI, it doesn't show them; that's odd